x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_add_edit/form/field_config.tsx

```diff
@@ -1251,4 +1281,168 @@ export const FIELD = (readOnly?: boolean): FieldMap => ({
       isDisabled: readOnly,
     }),
   },
+  [ConfigKey.MODE]: {
+    fieldKey: ConfigKey.MODE,
+    component: Select,
+    label: i18n.translate('xpack.synthetics.monitorConfig.mode.label', {
+      defaultMessage: 'Mode',
+    }),
+    helpText: (
+      <FormattedMessage
+        id="xpack.synthetics.monitorConfig.syntheticsArgs.mode.helpText"
+        defaultMessage="If mode is {any}, the monitor pings only one IP address for a hostname. If mode is {all}, the monitor pings all resolvable IPs for a hostname. The mode: {all} setting is useful if you are using a DNS-load balancer and want to ping every IP address for the specified hostname."
```

Comment:
```suggestion
        defaultMessage="If {any}, the monitor pings only one IP address for a hostname. If {all}, the monitor pings all resolvable IPs for a hostname. {all} is useful if you are using a DNS-load balancer and want to ping every IP address for the specified hostname."
```

This is rather long  even if there are only 2 possible values, so if time allows I would suggest to turn this into a reference popover, with a simple 2 columns table (value, description), and 1 row per value.
The popover could be open from an icon placed next to the setting label. 

Or maybe for a later improvement. I could also do it later because I need to practice implementing content improvements :)